src/renderer/routes/test_sequencer_panel/components/dnd/DraggableRowStep.tsx

```diff
@@ -71,10 +75,34 @@ export const DraggableRowStep = ({
     useConfigureDropRef(parseInt(row.id));
   const isActiveAbove = isOverAbove && canDropAbove;
 
+  let variant = "default" as
+    | "aborted"
+    | "pass"
+    | "fail"
+    | "running"
+    | "paused"
+    | "default";
+  if (row.original.type === "test") {
+    if (row.original.status !== "pending") {
+      variant = row.original.status as
```

Comment:
@39bytes, I'm not sure about `as`, but it's the best way I've found to convert a status to a variant. + there's a check to convert to default if the value doesn't exist (and Typescript will complain if a new status is added).
Curious if you know of any TS magic for this?

Reply:
> I'm confused as to why the .includes check is there when the type of row.original.status inside the if block is "aborted" | "pass" | "fail" | "running" | "paused"? 

Rigth now, it's never hit, it's only for the linter (+ let's say you add something else, the runtime don't panick, but typescript will tell us, so it's not actually needed). That's why I'm asking.

I'm testing your solutions, thanks

Reply:
```
  type Variant = Exclude<StatusType, "pending"> | "default";
  let variant: Variant = "default";
  if (row.original.type === "test") {
    if (row.original.status !== "pending") {
      variant = row.original.status;
    }
  }
```
The Exclude is perfect, + Typescript signals us that something is wrong when adding a new value in StatusType. Thanks